test/fixture/report/regular/test.js

```diff
@@ -26,8 +26,8 @@ test('formatted', t => {
 });
 
 test('power-assert', t => {
-	const foo = 'bar';
-	t.falsy(foo);
+	const foo = '';
+	t.assert(foo);
```

Comment:
To match the original test, shouldn't this be:

```js
t.assert(!foo);
```

Reply:
Don't really know. Is it important that they match?

Reply:
I guess @chocolateboy's point is that you also changed the value of `foo`. But, in the end, the test still fails, so it doesn't change too much.